interview programing part complete

Craigcanepa10 opened the pull request:
used an existing service to show the contents of the repository.

Commit:
interview programing part complete

used an existing service to show the contants of  the repository

Craigcanepa10 committed:
Edited image height

edited the images height down

dvicknair commented on Controllers/HomeController.cs:
```diff
@@ -17,9 +22,12 @@ public HomeController(ILogger<HomeController> logger, WidgetService widgetServic
         public IActionResult Index()
         {
-            return View();
+            object getAllWidgets = _widgetService.GetAllWidgets();
```
The variable "getAllWidgets" here is assigned the list of widgets returned by the function.  The next line sets the "list" variable equal to the same.  You should eliminate the "getAllWidgets" variable and assign the list returned by the function directly to the "list" variable on line 29.

Craigcanepa10 replied:
Removed the redundant variable and renamed getallwidgets to list.

dvicknair replied:
This looks great now!

dvicknair commented on Views/Home/Index.cshtml:
```diff
@@ -1,7 +1,51 @@
 @{
     ViewData["Title"] = "Home Page";
 }
+<head>
```
The head element located within the body of the html document is malformed html.  The head element should be at the sibling level and appear before the body element.

Craigcanepa10 replied:
added in correct HTML tags, sorry I overlooked those being missing.

dvicknair replied:
Please locate the _Layout.cshtml file in the solution.  You will notice the root level elements here.  The framework loads all other html elements created in the app inside the body element found in this file.  You will also find the head element in this file, and this would be the well-formed area to locate your styles.

Craigcanepa10 replied:
Ah the pages act as component files and _layout is the master file.

Craigcanepa10 committed:
fixed HTML and controller

fixed malfrormed HTML adding in the body and HTML tag, and removed the extra var that was not needed.

Craigcanepa10 committed:
renamed getallwidgets var

renamed getallwidgets var to list

Craigcanepa10 committed:
fixed issue and moved to correct folder
